workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/EmptyState/EmptyState.tsx

```diff
@@ -36,7 +36,7 @@ export const States = {
 	 * The user has emptied the dataset for a good cause.
 	 * For example, all the notifications have been addressed, resulting in a clean state.
 	 */
-	SUCCESS: `${Liferay.ThemeDisplay.getPathThemeImages()}/states/success_state.gif`,
+	SUCCESS: `${Liferay.ThemeDisplay.getPathThemeImages()}/states/success_state.svg`,
```

Comment:
Actually, it's probably just better for us not to make changes to the workspaces at all. They can also be using older versions of Clay components there, so in general it's not as safe of a location to make updates to.

Reply:
Thanks @ethib137, I'll remove them. 
Nevertheless, should I send them a message so they can be aware of the changes we are making through DXP?

Reply:
It wouldn't hurt to send them a message letting them know. Switching to SVG isn't a very important change for them to make though. The more important change is having a reduced motion image, since that provides accessibility benefits.